include/openssl/base.h

```diff
@@ -208,7 +208,7 @@ extern "C" {
 // against multiple revisions of BoringSSL at the same time. It is not
 // recommended to do so for longer than is necessary.
 
-#define AWSLC_API_VERSION 20
+#define AWSLC_API_VERSION 21
```

Comment:
Curious why its not possible to use `AWSLC_VERSION_NUMBER_STRING` to detect version. Is that field only meant for internal use?

Reply:
I'm sure it's _possible_, but we'd need to parse it and convert each ordinal to an integer in order to compare it. It seems more direct to use the integer-valued API version.